### DISCUSSION TITLE
Madtracker / MT2: how to play back these files properly

### DISCUSSION THREAD

**Discussion by ingejanse** [schismtracker/schismtracker#496] [Q&A] [answered]
Hi there,

I'm not a coder, just a user. I found some old files that I made with Madtracker in MT2 format. After a perpetual search on the internet, I stumbled upon Schismtracker. Looks very promising, it actually loads the files and shows the module info, but I can hear nothing and see no entries in the actual tracker interface. Is this a known bug, or is there something I should tinker with the settings to get this rolling? Thanks in advance! 

(I used to work for Madtracker, helping developer Yannick Delwiche out, so I have some fond memories regarding these mt2 files)

**Comment by mrpapersonic** [comment, marked as answer]
MT2 import is not supported at all, only getting module info is implemented. I doubt most will even play right in schism anyway since IIRC its based off of XM which already doesn't play correctly.

OpenMPT does have some support for the format but I can't say for how accurate the playback will be. For best results really you should just play them back in the original madtracker anyway.

**Reply by ingejanse** [reply]
Thanks, will find a Windows computer somewhere and see if I can install MT2 on it. Thanks for clarifying this!

**Reply by sagamusix** [reply]
If you need absolutely accurate playback, MT2 itself is still the only way to go, especially if DSP effects and plugins are involved. Otherwise, OpenMPT's MT2 support is fairly decent (mostly missing the built-in DSPs and automation curve support) - stuff made with older MT2 versions or not using any plugins will usually play fine though.
